src/ipyvizzustory/storylib/story.py

```diff
@@ -197,11 +258,52 @@ def is_pixel(value: Any) -> bool:
             `True` if the value is pixel, `False` otherwise.
         """
 
-        value_is_pixel = False
-        if isinstance(value, str):
-            if value.endswith("px"):
-                value_is_pixel = value[:-2].isnumeric()
-        return value_is_pixel
+        if StorySize._is_int(value) or StorySize._is_float(value):
+            return True
+        if isinstance(value, str) and value.endswith("px"):
+            if StorySize._is_int(value[0:-2]) or StorySize._is_float(value[0:-2]):
+                return True
+        return False
+
+    def get_width_height_in_pixels(self) -> Tuple[int, int]:
+        """
+        A method for returning the width and height in pixels.
+
+        Raises:
+            ValueError: If width and height are not in pixels when aspect_ratio is not set.
+            ValueError: If width or height is not in pixel when aspect_ratio is set.
+            ValueError: If aspect_ratio is not a float when aspect_ratio is set.
+
+        Returns:
+            The width and height in pixels as int.
+        """
+
+        if self.aspect_ratio is None:
+            if any(
+                [
+                    not StorySize.is_pixel(self.width),
+                    not StorySize.is_pixel(self.height),
+                ]
+            ):
+                raise ValueError("width and height should be in pixels")
+            _width = int(float(self.width[:-2]))  # type: ignore
+            _height = int(float(self.height[:-2]))  # type: ignore
+        else:
+            if not isinstance(self.aspect_ratio, (float, int)):
```

Comment:
I think there is a weird inconsistent working. When we set:

`StorySize(1, 1, 0.5)` is OK but
`StorySize('1', '1', '0.5')` is not, only because the aspect ratio doesn't accept numbers as string while other arguments are fine with that.

Reply:
fixed in [0952c88](https://github.com/vizzuhq/ipyvizzu-story/pull/75/commits/0952c88d8d4b26d411daf74674d61c7c01e7236a)